packages/core/src/components/steps-modal/steps-modal.tsx

```diff
@@ -0,0 +1,100 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  Host,
+  Prop,
+  forceUpdate,
+  h,
+} from '@stencil/core'
+
+@Component({
+  tag: 'atom-steps-modal',
+  styleUrl: 'steps-modal.scss',
+  shadow: false,
```

Comment:
question: why `shadow: false`? it's removing the component scope of CSS

Reply:
that was made because of when we use atom-modal inside of this component, the shadow scope removes the style from it, leaving the modal without any style
![image](https://github.com/user-attachments/assets/64783c64-7684-4fad-83c4-a2254b9279df)